### PR TITLE
UX: Number formatting, new layout for routes, smaller IP switch

### PR DESCRIPTION
**Pull request by ai-slave**
image

### CODE REVIEW

**Comment by 0xngmi** [comment]
image
smol bug when gas cost is unknown

**Comment by 0xngmi** [comment]
unsure about adding price below each quote since it seems a bit redundant, whats the rationale behind that?

**Comment by ai-slave** [comment]
@0xngmi to compare what was the initial quote from the aggregator. But maybe a good place to put "~ $9,900 After fees"?

**Comment by ai-slave** [comment]
@0xngmi updated to this version 


Price is being shown in #50, I'll maybe iterate a little later to incorporate highlighting high impact orders on the list of routes, so you can understand it's not good before you click on it.

**Comment by ai-slave** [comment]
removed .toFixed(3) on final amount that you're getting
image

**Comment by ai-slave** [comment]
Anything else weird left on this one?

**Comment by 0xngmi** [comment]
looks good, will review in a bit again and ask for more reviews

**Comment by 0xngmi** [comment]
sorry but recently merged a PR that made a lot of changes, could you fix the merge conflicts?

**Comment by ai-slave** [comment]
@0xngmi done

**Review by mintdart — CHANGES_REQUESTED** [review]
can you update this row color, it is a bit harder to read
Screenshot 1

**Review by mintdart — DISMISSED** [review]

**Comment by mintdart** [comment]
> can you update this row color, it is a bit harder to read Screenshot 1

Please ignore this, I've changed it in my last commit

**Comment by mintdart** [comment]
on mobile
Screenshot

**Comment by 0xngmi** [comment]
image
z-index issue

**Comment by ai-slave** [comment]
@0xngmi can't reproduce, but topped zindex, please check

**Comment by 0xngmi** [comment]
still seeing same issue, on prod at https://llamaswap-git-fork-ai-slave-master-llamapay.vercel.app/ (tested with brave and firefox)

**Comment by ai-slave** [comment]
@0xngmi should be fixed

**Comment by mintdart** [comment]
I still see z-index issue on https://llamaswap-git-fork-ai-slave-master-llamapay.vercel.app/?from=0x0000000000000000000000000000000000000000&to=0x4d224452801aced8b2f0aebe155379bb5d594381
Screenshot

**Comment by ai-slave** [comment]
Fixed, thank you guys for testing it. I'll try to do better :|

**Comment by mintdart** [comment]
> Fixed, thank you guys for testing it. I'll try to do better :|

If z-index is -1, you can't select any route

**Comment by ai-slave** [comment]
fixed, layers should be fine now

**Comment by mintdart** [comment]
on mobile :|
Screenshot

**Comment by ai-slave** [comment]
fixed 👼 thank you

**Comment by 0xngmi** [comment]
your fix might not have been pushed because of mintdart's commit, after which you need to rebase

**Comment by ai-slave** [comment]
yeah sorry, just forgot to push :/

File: src/utils/index.tsx [hunk 1]
```diff
@@ -15,3 +15,41 @@ export const median = (arr: number[]): number => {
...
+	let currencySymbol;
+	if (symbol === true) {
+		currencySymbol = '$';
+	} else if (symbol === false) {
+		currencySymbol = '';
+	} else {
+		currencySymbol = symbol;
+	}
+	if (!number || number === '' || Number.isNaN(Number(number))) {
+		return symbol ? `${currencySymbol}0` : 0;
```
Comment (0xngmi) [inline src/utils/index.tsx:29]: wouldnt it be better to properly fix these cases to report 0 instead of something weird?

**Review by 0xngmi — APPROVED** [review]
minimal stuff, good to merge

File: src/utils/index.tsx (hunk 1 above)
Comment (ai-slave) [reply src/utils/index.tsx:29]: I got this function from Llama main dashboard, but the use case changed a little so yeah, I'll need to rewrite it and take off junk

**Comment by 0xngmi** [comment]
new ui bussin frfr

File: src/utils/index.tsx (hunk 1 above)
Comment (0xngmi) [reply src/utils/index.tsx:29]: imo already good to merge, gonna wait for minty's greenlight and merge

**Review by mintdart — APPROVED** [review]

**Pull request merged**